zad3/app/controllers/bestiary_controller.rb

```diff
@@ -2,15 +2,20 @@
 
 class BestiaryController < ApplicationController
   def index
-
+    monsters
   end
 
   def novigrad_monsters
-
+    @monsters = monsters.select(&from?("novigrad"))
+    render 'index'
```

Comment:
Sprytnie 👍 🎉 Jak podszedłbyś do takiego nowego wymagania?:
- Strona z potworami z Novigradu powinna dodatkowo, ponad tabelą, wyświetlać info o tym, że są to potwory z Novigradu?